roles/test_operator/tasks/run-test-operator-job.yml

```diff
@@ -67,15 +67,43 @@
     - not cifmw_test_operator_dry_run | bool
     - not testjob_timed_out
   block:
-    - name: Get information about PVC that stores the logs
+    - name: Get information about PVCs that store the logs
       kubernetes.core.k8s_info:
         kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
         api_key: "{{ cifmw_openshift_token | default(omit)}}"
         context: "{{ cifmw_openshift_context | default(omit)}}"
         kind: PersistentVolumeClaim
         label_selectors:
           - "instanceName={{ test_operator_job_name }}"
-      register: logsPVC
+      register: logsPVCs
+
+    - name: Set up empty volume mounts and volumes
+      set_fact:
+        volume_mounts: []
+        volumes: []
+
+    - name: Loop through PVCs and set up volume mounts and volumes
+      set_fact:
+        volume_mounts: >
+          {{
+            volume_mounts + [{
+              'name': 'logs-volume-' ~ index,
+              'mountPath': '/mnt/logs-volume-' ~ index
```

Comment:
Also, can we change the name of the logs folder to something like this?
`'/mnt/{{ test_operator_job_name }}-step-' ~ index`

Reply:
If you would be able to find a way how to put step name into the folder name that would be even better:).

Reply:
I left it at adding only test_operator_job_name as I have been stuck on this for a little bit too long :(